api/config/config_perf.json

```diff
@@ -106,7 +106,9 @@
     "enableCohortBuilderV2": true,
     "enableConceptSetSearchV2": true,
     "enableReportingUploadCron": false,
-    "enableCustomRuntimes": true
+    "enableCustomRuntimes": true,
+    "bufferBillingProjectsInPerimeter": true,
```

Comment:
I expect `false` to have significant performance impact.  What does that imply for the perf env?